### PR TITLE
[CLEANUP] Fix Scrutinizer complaints in event dispatcher

### PR DESCRIPTION
**Pull request by clement-michelet**
See: https://scrutinizer-ci.com/g/itkg/delay-event-bundle/inspections/e4c73d0e-87f3-4afc-8aa6-b9d1857497be/issues/files/EventDispatcher/DelayEventDispatcher.php?status=new&orderField=path&order=asc&honorSelectedPaths=0

### CODE REVIEW

File: EventDispatcher/DelayEventDispatcher.php [hunk 1]
```diff
@@ -102,7 +107,7 @@ public function hasListeners($eventName = null)
     /**
      * Proxies all method calls to the original event dispatcher.
      *
-     * @param string $method    The method name
+     * @param string $method The method name
      * @param array  $arguments The method arguments
```
Comment (juchi) [inline EventDispatcher/DelayEventDispatcher.php:111]: i believe the previous formatting was correct

**Comment by clement-michelet** [comment]
ping @juchi

**Comment by juchi** [comment]
:+1:

**Comment by pdenis** [comment]
:+1:

**Pull request merged**